### ISSUE TITLE
Alert: refactor queue timeout to get timeout value from the CSS var `--calcite-internal-animation-timing-slow`

### ISSUE THREAD

**Issue by driskull** [Esri/calcite-design-system#10098]
**monday.com sync:** #18048700142

### Description

Currently, the alert tests need to set a fixed timeout to account for animations even when animations are disabled. This is because the alert uses a fixed timeout for 300ms to wait for an animation to complete.

### Proposed Advantages

Instead, it should get the timeout MS value from the animation CSS variable.

Then, in tests `await page.waitForTimeout(animationDurationInMs);` should be able to be removed.

### Which Component

alert

### Relevant Info

https://github.com/Esri/calcite-design-system/pull/10029#discussion_r1720467890

### Calcite package

- [X] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

**Comment by driskull** [comment]
Looked into this a bit. One issue is that the CSS variable `--calcite-internal-animation-timing-slow` uses `calc`.

Since it uses calc, we can't get the calculated value in order to find out the real timeout. That might make this a bit tough.

**Comment by jcfranco** [comment]
Related: https://github.com/Esri/calcite-design-system/issues/6793.

**Comment by isaacbraun** [comment]
#6793 was closed as duplicate of #6395. Would the proposed changes to `autoCloseDuration` in #6395 resolve this queue timeout testing issue?

cc @driskull

**Comment by driskull** [comment]
Yes, if we can have a standard value that can be parsed.

**Comment by isaacbraun** [comment]
Gotcha, thanks! Marking this as blocked by the efforts of #6395 then.
